internal/services/paloalto/schema/marketplace_details.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
Suggest adding these new properties to [resource docs](https://github.com/hashicorp/terraform-provider-azurerm/tree/main/website/docs/r). Thanks.